### PR TITLE
CAMEL-19686 - Remove Camel-Atlasmap

### PR DESCRIPTION
**Pull request by oscerd**
# Description

# Target

- [x] I checked that the commit is targeting the correct branch (note that Camel 3 uses `camel-3.x`, whereas Camel 4 uses the `main` branch)

# Tracking
- [ ] If this is a large change, bug fix, or code improvement, I checked there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).

# Apache Camel coding standards and style

- [x] I checked that each commit in the pull request has a meaningful subject line and body.

- [x] I have run `mvn clean install -DskipTests` locally and I have committed all auto-generated changes

### CODE REVIEW

File: components/camel-attachments/src/test/java/org/apache/camel/attachment/MulticastAggregationStrategyTest.java
Comment (essobedo) [inline components/camel-attachments/src/test/java/org/apache/camel/attachment/MulticastAggregationStrategyTest.java:1]: Doesn't seem to be related

File: core/camel-support/src/main/java/org/apache/camel/support/ExchangeHelper.java
Comment (essobedo) [inline core/camel-support/src/main/java/org/apache/camel/support/ExchangeHelper.java:1]: Doesn't seem to be releated

**Review by essobedo — APPROVED** [review]

File: components/camel-attachments/src/test/java/org/apache/camel/attachment/MulticastAggregationStrategyTest.java
Comment (oscerd) [reply components/camel-attachments/src/test/java/org/apache/camel/attachment/MulticastAggregationStrategyTest.java:1]: running a full build will add the license, so I suppose it's fine.

File: core/camel-support/src/main/java/org/apache/camel/support/ExchangeHelper.java
Comment (oscerd) [reply core/camel-support/src/main/java/org/apache/camel/support/ExchangeHelper.java:1]: Same

**Review by davsclaus — APPROVED** [review]
